modules/targets/generic-linux.nix

```diff
@@ -32,10 +32,17 @@ in {
       dataDirs = concatStringsSep ":"
         (map (profile: "${profile}/share") profiles
           ++ config.targets.genericLinux.extraXdgDataDirs);
-    in { XDG_DATA_DIRS = "${dataDirs}\${XDG_DATA_DIRS:+:}$XDG_DATA_DIRS"; };
+    in {
+      XDG_DATA_DIRS = "${dataDirs}\${XDG_DATA_DIRS:+:}$XDG_DATA_DIRS";
+      TERMINFO_DIRS =
+        "${profileDirectory}/share/terminfo:$TERMINFO_DIRS\${TERMINFO_DIRS:+:}/usr/share/terminfo";
```

Comment:
Besides `/usr/share/terminfo`, are there other possible values?

Reply:
I've now looked at package definitions from other distros and added `/etc/terminfo` and `/lib/terminfo`. `/usr/share/terminfo` is the default lookup location defined by ncurses.